src/github.com/getlantern/flashlight/analytics/analytics.go

```diff
@@ -25,7 +27,8 @@ const (
 )
 
 var (
-	log = golog.LoggerFor("flashlight.analytics")
+	log  = golog.LoggerFor("flashlight.analytics")
+	elog = errlog.ErrorLoggerFor("flashlight.analytics")
```

Comment:
@fffw I like your idea of enhancing golog to handle special reporting of errors.  Since the reporting logic probably just goes into the Reporter implementation, we could actually make this opaque to golog by just using the built-in error type.  For example:

```go
package golog

...

type Reporter interface {
    Report(error)
}

func (l *logger) Error(arg interface{}) {
    // Log just as before
    l.print(GetOutputs().ErrorOut, 4, "ERROR", arg)

    if l.reporter != nil {
        err, isError := arg.(error)
        if isError {
            l.reporter.Report(err)
        }
    }
}
```